src/DependencyInjection/Kfz24QueueExtension.php

```diff
@@ -20,21 +20,21 @@ class Kfz24QueueExtension extends Extension
 {
     /**
      * {@inheritdoc}
+     * @throws \Exception
      */
     public function load(array $configs, ContainerBuilder $container)
     {
         $configuration = new Configuration();
         $config = $this->processConfiguration($configuration, $configs);
 
-        $loader = new Loader\XmlFileLoader($container, new FileLocator(__DIR__ . '/../Resources/config'));
-        $loader->load('services.xml');
+        $loader = new Loader\YamlFileLoader($container, new FileLocator(__DIR__ . '/../Resources/config'));
+        $loader->load('services.yaml');
 
         foreach ($config['clients'] as $name => $client) {
             $clientType = $client['type'];
-
+            $clientClass = $container->getParameter(sprintf('kfz24.queue.%s.client.class', $clientType));
```

Comment:
I think the order here used to be alphabetically? How come to change it?

Reply:
Artifact of back and forth changes. I'll revert it